Packetery/Checkout/Block/System/Config/Form/ApiKeyValidate.php

```diff
@@ -1,28 +1,26 @@
 <?php
 
-
 namespace Packetery\Checkout\Block\System\Config\Form;
 
-
+use Magento\Framework\App\Config\Value;
+use Magento\Framework\Exception\ValidatorException;
 use Magento\Framework\Phrase;
 
-class ApiKeyValidate extends \Magento\Framework\App\Config\Value
+class ApiKeyValidate extends Value
 {
-    const API_KEY_LENGTH = 16;
+    private const API_KEY_LENGTH = 16;
 
     /**
-     * @return void
-     * @throws \Magento\Framework\Exception\ValidatorException
-     * @throws \Magento\Framework\Exception\FileSystemException
+     * Validates API key.
+     *
+     * @throws ValidatorException
```

Comment:
Proč jsi zkracoval ? Zatím jsem nezaznamenal žádnou diskusi, že  místo celého namespacu začneme zkracovat. Zatím jsou to za mne zbytečné změny, které se vyřeší až nasazením nového coding stylu.